File: Navtrack.Core/ClientApp/src/apis/types/user/UserModel.ts
import { MeasurementSystem } from "../general/MeasurementSystem";

export type UserModel = {
  id: number;
  email: string;
  measurementSystem: MeasurementSystem;
};

export const DefaultUserModel: UserModel = {
  id: 0,
  email: "",
  measurementSystem: MeasurementSystem.Metric
};

File: Navtrack.Api.Model/User/UserModel.cs
using Navtrack.Api.Model.Custom;
using Navtrack.Api.Model.General;

namespace Navtrack.Api.Model.User
{
    public class UserModel : IModel
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public MeasurementSystem MeasurementSystem { get; set; }
    }
}